src/test-helpers/contextToThought.ts

```diff
@@ -9,7 +9,7 @@ import getThoughtById from '../selectors/getThoughtById'
  */
 const contextToThought = (state: State, context: Context): Thought | null => {
   const id = contextToThoughtId(state, context)
-  return id ? getThoughtById(state, id) : null
+  return id ? (getThoughtById(state, id) ?? null) : null
```

Comment:
to preserve type for now

but there's an itch to make it `Thought | null | undefined`
wdyt?